.mvn/maven-build-cache-config.xml

```diff
@@ -21,9 +21,9 @@
     <!--
         Maven build cache configuration for aiSSEMBLE
 
-        This feature requires Maven 3.9+ and is currently disabled by default while we work out the kinks. To use it,
-        you must pass the -Dmaven.build.cache.enabled=true flag to your build. It is also useful to look over the other
-        build options available: https://maven.apache.org/extensions/maven-build-cache-extension/parameters.html
+        This feature requires Maven 3.9+, and is enabled by default. To disable it, you must pass the
```

Comment:
A(Administrative): I think we want to steer people to use `skipCache` as the flag (does not use the cache during the build, but still stores the output to improve cache hits later).  If they want to completely disable it, then updating `<enabled>` to be false in this file is the best move.

Reply:
Makes sense.  Change has been made in both locations.